Trimble.Modus.Components/Controls/TMChips/TMChips.xaml.cs

```diff
@@ -0,0 +1,319 @@
+namespace Trimble.Modus.Components;
+
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Shapes;
+using System;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Enums;
+using Trimble.Modus.Components.Helpers;
+
+public partial class TMChips : ContentView
+{
```

Comment:
Selected color is not right, It should be light blue opaque color. But here it is our Primary Trimble Blue color
<img width="529" alt="selected-color" src="https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/1e834622-21c3-413f-9f4d-d801d0feaf50">

Reply:
Its actually 30 percent of TrimbleBlue Color so added it as a new color for chip selected

Reply:
Looks like it is fixed now?

Reply:
Yes